node_modules/col-assets/lib/api.js

```diff
@@ -144,6 +147,14 @@ var getAsset = module.exports.getAsset = function(ctx, id, opts, callback) {
       'required': false
     });
   }
+  if (opts.includeWhiteboardUsages) {
```

Comment:
Ah, that's unfortunate. Yeah, it's probably easiest to add a column. Note that we'll then also need to create a migration script for the existing data.